components/dashboard/src/Menu.tsx

```diff
@@ -180,55 +186,45 @@ export default function Menu() {
             ];
         }
         // Team menu
-        if (team) {
-            const currentUserInTeam = (teamMembers[team.id] || []).find((m) => m.userId === user?.id);
-
-            const teamSettingsList = [
-                {
-                    title: "Projects",
-                    link: `/t/${team.slug}/projects`,
-                    alternatives: [] as string[],
-                },
-                {
-                    title: "Members",
-                    link: `/t/${team.slug}/members`,
-                },
-            ];
-            if (showUsageView || (teamBillingMode && teamBillingMode.mode === "usage-based")) {
-                teamSettingsList.push({
-                    title: "Usage",
-                    link: `/t/${team.slug}/usage`,
-                });
-            }
-            if (currentUserInTeam?.role === "owner") {
-                teamSettingsList.push({
-                    title: "Settings",
-                    link: `/t/${team.slug}/settings`,
-                    alternatives: getTeamSettingsMenu({ team, billingMode: teamBillingMode }).flatMap((e) => e.link),
-                });
-            }
-
-            return teamSettingsList;
+        if (!team) {
+            return [];
         }
-        // User menu
-        const userMenu = [];
-        userMenu.push({
-            title: "Projects",
-            link: "/projects",
-        });
-        if (showUsageView) {
-            userMenu.push({
+        const currentUserInTeam = (teamMembers[team.id] || []).find((m) => m.userId === user?.id);
+
+        const teamSettingsList = [
+            {
+                title: "Projects",
+                link: `/t/${team.slug}/projects`,
+                alternatives: [] as string[],
+            },
+            {
+                title: "Members",
+                link: `/t/${team.slug}/members`,
+            },
+        ];
+        if (showUsageView || (teamBillingMode && teamBillingMode.mode === "usage-based")) {
+            teamSettingsList.push({
                 title: "Usage",
-                link: "/usage",
+                link: `/t/${team.slug}/usage`,
             });
         }
-        userMenu.push({
-            title: "Settings",
-            link: "/settings",
-            alternatives: getSettingsMenu({ userBillingMode }).flatMap((e) => e.link),
-        });
-        return userMenu;
+        if (currentUserInTeam?.role === "owner") {
+            teamSettingsList.push({
+                title: "Settings",
+                link: `/t/${team.slug}/settings`,
+                alternatives: getTeamSettingsMenu({ team, billingMode: teamBillingMode }).flatMap((e) => e.link),
+            });
+        }
+
+        return teamSettingsList;
     })();
+    const leftMenu: Entry[] = [
+        {
+            title: "Workspaces",
```

Comment:
**thought**: Cross-posting an alternative and less abrupt approach for future reference, which be keep the global workspaces list on the right and use the top left pills for sub-entities like Projects, etc but also for the project path. However, let's keep this discussion separate and out of the scope of the changes here.

| Before creating a team | After creating a team |
|-|-|
| <img width="1440" alt="Screenshot 2022-09-29 at 2 37" src="https://user-images.githubusercontent.com/120486/193856823-1ad507a6-6458-43b4-a286-e6add1b12c9a.png"> | <img width="1440" alt="Screenshot 2022-09-29 at 2 41" src="https://user-images.githubusercontent.com/120486/193856848-ea940f8c-7361-4de0-9120-5a4cab94fcda.png"> |